### PR TITLE
fixes search style transitions on hover/active

### PR DESCRIPTION
**Pull request by astewes**
*  Keeps search bar open if user has entered text and then mouses away
*  Search bar now opens from the search icon and closes toward it

searchbar

### CODE REVIEW

**Comment by nitinmotgi** [comment]
can we change "Search Tags" to "Search Metadata" - Make it generic.

**Comment by elfenheart** [comment]
Can we clear out the searchText once we hit enter?

File: cdap-ui/app/directives/search/search-ctrl.js [hunk 1]
```diff
@@ -22,11 +22,13 @@ angular.module(PKG.name + '.commons')
     this.onSearch = function(event) {
       if (event.keyCode === 13) {
         $state.go('search.objectswithtags', {tag: this.searchTxt});
+        this.searchTxt = '';
```
Comment (elfenheart) [inline cdap-ui/app/directives/search/search-ctrl.js]: you have to set this.displaySearchBox to false as well, otherwise the box won't close

**Comment by elfenheart** [comment]
:+1: LGTM

**Pull request merged**